### ISSUE TITLE
Feature Request: Add full support for API Data

### ISSUE THREAD

**Issue by esDotDev** [lorenzocopelli/weatherapi#2]
The `Hour` data in this pkg support only `tempC`, `time` and `condition`, while the Weather API returns much more:
```json
{
    "time_epoch": 1715209200,
    "time": "2024-05-09 00:00",
    "temp_c": 11.6,
    "temp_f": 52.9,
    "is_day": 0,
    "condition": {
        "text": "Cloudy ",
        "icon": "//cdn.weatherapi.com/weather/64x64/night/119.png",
        "code": 1006
    },
    "wind_mph": 3.6,
    "wind_kph": 5.8,
    "wind_degree": 187,
    "wind_dir": "S",
    "pressure_mb": 1029.0,
    "pressure_in": 30.4,
    "precip_mm": 0.0,
    "precip_in": 0.0,
    "snow_cm": 0.0,
    "humidity": 89,
    "cloud": 63,
    "feelslike_c": 11.4,
    "feelslike_f": 52.5,
    "windchill_c": 11.4,
    "windchill_f": 52.5,
    "heatindex_c": 11.6,
    "heatindex_f": 52.9,
    "dewpoint_c": 9.9,
    "dewpoint_f": 49.7,
    "will_it_rain": 0,
    "chance_of_rain": 0,
    "will_it_snow": 0,
    "chance_of_snow": 0,
    "vis_km": 10.0,
    "vis_miles": 6.0,
    "gust_mph": 6.0,
    "gust_kph": 9.7,
    "uv": 1.0
},
```

Could you add support for these additional fields?

**Comment by lorenzocopelli** [comment]
Sure @esDotDev, I will add them as soon as possible.

**Comment by esDotDev** [comment]
Thanks! Also please note the additional properties on Condition, `text` and `code`

**Comment by lorenzocopelli** [comment]
Hi @esDotDev . I've released version 1.1.0, which includes support for all API data fields. Let me know if there are any issues.

**Comment by esDotDev** [comment]
This is working great, ty!
